pkg/cmd/cli/debug/cshd-scripts/velero.cshd

```diff
@@ -0,0 +1,25 @@
+def capture_backup_logs(namespace):
+    if args.backup:
+        kube_capture(what="objects", namespaces=[namespace], groups=['velero.io'], kinds=['backup'], names=[args.backup])
+        backupLogsCmd = "velero --namespace={} backup logs {}".format(namespace, args.backup)
+        capture_local(cmd=backupLogsCmd, file_name="backup_{}.log".format(args.backup))
+def capture_restore_logs(namespace):
+    if args.restore:
+        kube_capture(what="objects", namespaces=[namespace], groups=['velero.io'], kinds=['restore'], names=[args.restore])
+        restoreLogsCmd = "velero --namespace={} restore logs {}".format(namespace, args.restore)
+        capture_local(cmd=restoreLogsCmd, file_name="restore_{}.log".format(args.restore))
+
+ns = args.namespace if args.namespace else "velero"
+basedir = args.basedir if args.basedir else os.home
+output = args.output if args.output else "bundle.tar.gz"
+# Working dir for writing during script execution
+crshd = crashd_config(workdir="{0}/velero-bundle".format(basedir))
+set_defaults(kube_config(path=args.kubeconfig))
+capture_local(cmd="velero version -n {}".format(ns), file_name="version.txt")
+capture_backup_logs(ns)
+capture_restore_logs(ns)
+kube_capture(what="logs", namespaces=[ns])
+kube_capture(what="objects", namespaces=[ns], groups=['velero.io'], kinds=['backupstoragelocation', 'podvolumebackup', 'podvolumerestore'])
```

Comment:
I think we should also grab:
resticrepositories
schedules
volumesnapshotlocations
downloadrequests
deletebackuprequests

We're grabbing podvolumebackup and podvolumerestore here but we only capture the backup/restore resources if the backup or restore flag is selected.  Resources are pretty small, can we just capture all of the velero.io resources here regardless of whether they've selected a particular backup or restore?  Backup/restor log files are big, and we need to retrieve from object storage so it sounds right to only include those if people specify a particular backup/restore.

Reply:
Updated the crashd script to collect all resources in group `velero.io`